control-plane/pkg/reconciler/sink/controller.go

```diff
@@ -70,5 +73,14 @@ func NewController(ctx context.Context, _ configmap.Watcher, configs *config.Env
 
 	sinkInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
 
+	globalResync := func(obj interface{}) {
+		impl.GlobalResync(sinkInformer.Informer())
+	}
+
+	configmapInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
```

Comment:
Should this be filtered only by delete events?

Reply:
Add and Delete, no Update